CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ set(CMAKE_AUTORCC ON)
 project(QtAdvancedDockingSystem VERSION ${ads_VERSION})
 option(BUILD_STATIC "Build the static library" OFF)
 option(BUILD_EXAMPLES "Build the examples" ON)
-set(REQUIRED_QT_VERSION 5.0.0)
+set(REQUIRED_QT_VERSION 5.7.0)
```

Comment:
Some days ago I merged a pull request with fixes to enable building for Qt 5.5.1 on Ubuntu 16.04. So what is the reason to require Qt 5.7.0 here?. I think this will break the previous fix.

Reply:
It was a guess, really. I was sure 5.0 didn't work but don't know what's the minimum supported version. Since you are forcing C++11 I guessed 5.7 but happy to change it if you have a precise minimum requirement

Reply:
Ok, then I would prefer to change it to Qt 5.5.0 to ensure it can be build on Ubuntu 16.04.

Reply:
Sure, do you want to test it to a minimum required Qt version? Travis with Ubuntu can test any version of Qt >Qt5.0

Reply:
I would say we use Qt 5.5.1.